src/libs/HttpUtils.js

```diff
@@ -28,10 +30,30 @@ function processHTTPRequest(url, method = 'get', body = null) {
  * @returns {Promise}
  */
 function xhr(command, data, type = CONST.NETWORK.METHOD.POST, shouldUseSecure = false) {
+    if (command !== 'Log') {
+        info('Making API request', false, {
+            command,
+            type,
+            shouldUseSecure,
+            rvl: data.returnValueList,
+        });
+    }
```

Comment:
Yes, you can see that, but this gives you a nice trace of what happened in the user's device and you can see everything that happened in one request, especially because of differing timestamps. Combining data from many requests is harder. So with this, you can see (not literal):
- Navigating to /r/65476394
- Making API request Get blablabla
- API request finished with code 407 and requestID XXXX
- Error thrown Bla!

And if you want to know why the API returned 407 you can search for that requestID.

Having said that, we will see how useful it is once we have it in prod and can use it.

Reply:
Got it. Thanks for the explanation! I didn't think about how we can use the requestID to find more logs.